pkg/util/util.go

```diff
@@ -48,3 +55,32 @@ func PrettyJSON(b []byte) []byte {
 	}
 	return out.Bytes()
 }
+
+// ReadFromDir reads a directory recursively and gets the files based on filter.
```

Comment:
Add: It returns a concatenated output of all matching files' content.

Reply:
ok will update